test/extended/util/client.go

```diff
@@ -123,7 +124,8 @@ func NewCLIWithoutNamespace(project string) *CLI {
 	client := &CLI{}
 
 	// must be registered before the e2e framework aftereach
-	g.AfterEach(client.TeardownProject)
+	// To do: Could use workspace instead of project for kcp test ?
+	// g.AfterEach(client.TeardownProject)
```

Comment:
yes, i think we should do that

Reply:
OK. I'll add it next PR for create a ws in the top beforeEach.